src/bp/ui-studio/src/web/views/OneFlow/sidePanel/form/SaySomethingForm.tsx

```diff
@@ -0,0 +1,244 @@
+import { Dropdown, MoreOptions, MoreOptionsItems, style as sharedStyle } from 'botpress/shared'
+import _ from 'lodash'
+import React, { FC, Fragment, useEffect, useReducer, useState } from 'react'
+import { connect } from 'react-redux'
+import {
+  closeFlowNodeProps,
+  copyFlowNode,
+  fetchContentCategories,
+  pasteFlowNode,
+  refreshFlowsLinks,
+  requestEditSkill,
+  updateFlow
+} from '~/actions'
+import ContentForm from '~/components/ContentForm'
+import { toastInfo } from '~/components/Shared/Utils'
+import withLanguage from '~/components/Util/withLanguage'
+import { getCurrentFlow, getCurrentFlowNode } from '~/reducers'
+import EditableInput from '~/views/FlowBuilder/common/EditableInput'
+
+import style from './style.scss'
+
+const { MoreOptionsStyles } = sharedStyle
+
+interface OwnProps {
+  onDeleteSelectedElements: () => void
+  readOnly: boolean
+  subflows: any
+  formData: any
+  updateNode: any
+  contentLang: string
+  defaultLanguage: string
+}
+
+type StateProps = ReturnType<typeof mapStateToProps>
+type DispatchProps = typeof mapDispatchToProps
+type Props = DispatchProps & StateProps & OwnProps
+
+export interface FormState {
+  contentType: string
+  error: any
+}
+
+const shownCategories = ['builtin_text', 'builtin_image', 'builtin_carousel', 'builtin_card']
```

Comment:
Content types have the property "hidden" which should be preferred instead of this hardcoded list, since i'm pretty sure we'll forget to update it here

Reply:
For now it's meant to be a patch for content types to still appear in the CMS, but not in the Say Node, this will be reworked in the future, but for now it'll be like that